capabilities_matrix/_topics/providers.md

```diff
@@ -17,7 +17,7 @@ The following table lists providers that {{ site.data.product.title_short }} can
 | [IBM PowerVC](#cloud-providers)                                                      |
 | IBM Terraform                                                                        |
 | [Kubernetes](#container-providers)                                                   |
-| [KubeVirt](#infrastructure-providers)                                                |
+| [OpenShift Virtualization (KubeVirt)](#infrastructure-providers)                     |
```

Comment:
In this case, please make these 2 separate line items.  That's what we've done for oVirt and Red Hat Virtualization.  Additionally, this particular screen should spell it out so, for OSV use the phrase `Red Hat OpenShift Virtualization (OSV)`, and then `KubeVirt` separately. (and of course in alphabetical order).